### ISSUE TITLE
Will not install on brand new MacBook Pro 15"

### ISSUE THREAD

**Issue by wereveal** [Homebrew/brew#2864]
# Please always follow these steps:
- [x ] Confirmed this is a problem with running a `brew` command and not `brew install`ing or the post-install behaviour of one or more formulae? If it's a formulae-specific problem please file this issue at https://github.com/Homebrew/homebrew-core/issues/new
- [x ] Ran `brew update` and retried your prior step?
- [ x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [ x
[brew.config.txt](https://github.com/Homebrew/brew/files/1126541/brew.config.txt)
[brew.doctor.txt](https://github.com/Homebrew/brew/files/1126544/brew.doctor.txt)

] Ran `brew config` and `brew doctor` and included their output with your issue?

**Please note we will close your issue without comment if you delete or do not fill out the issue checklist and provide ALL the requested information.**

To help us debug your issue please explain:
- What you were trying to do (and why)
   - Trying to install brew using the command on the Brew Homepage on a brand new MacBook Pro 15. 
   - Attempted this several times with same results. 
- What happened (include command output)
   - see the brew.doctor.txt file for basic results
   - basically trying to run any command including brew doctor except brew config results in the text output as seen in brew.doctor.txt 
- What you expected to happen
   - It to work. I have installed brew many times on many different Macs so this kind of surprised me.

- Step-by-step reproduction instructions (by running `brew` commands)
/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"

results = 
Error: Cannot tap homebrew/core: invalid syntax in tap!
Error: Failure while executing: /usr/local/bin/brew tap homebrew/core
Failed during: /usr/local/bin/brew update --force

**Comment by MikeMcQuaid** [comment]
Can you try to run `brew update --force` again?

**Comment by wereveal** [comment]
Same results as before. Last three lines of the error messages:
Error: Non-symlink alias: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Aliases/zmq
Error: Cannot tap homebrew/core: invalid syntax in tap!
Error: Failure while executing: /usr/local/bin/brew tap homebrew/core

**Comment by MikeMcQuaid** [comment]
What's in your `cat ~/.gitconfig`?

**Comment by wereveal** [comment]
Hmm, never thought about gitconfig being an issue. Have a fairly large .gitconfig.

**Comment by MikeMcQuaid** [comment]
Closing as this only affects you but will keep talking here.

**Comment by wereveal** [comment]
FYI: I temporarily move my .gitconfig file and brew installed without issue to that was the problem.

**Comment by MikeMcQuaid** [comment]
Cool, thought so. Things will break in future if it's still there; you have a setting enabled that's breaking symlinks so you may want to fix that.
